pkg/integrations/v4/manager_test.go

```diff
@@ -976,3 +984,9 @@ func expectNoMetric(t require.TestingT, e *testemit.Emitter, pluginName string)
 	_, err := e.ReceiveFrom(pluginName)
 	require.Error(t, err)
 }
+
+func skipIfWindows(t *testing.T) {
```

Comment:
wdyt about moving all these tests to a windows tagged file so they don't even get compiled there?

Reply:
Great comment, we will address it in further development.